docs/source/using-executorch-export.md

```diff
@@ -1,3 +1,173 @@
-# Model Export
+# Model Export and Lowering
 
-Placeholder for top-level export documentation
\ No newline at end of file
+The section describes the process of taking a PyTorch model and converting to the runtime format used by ExecuTorch. This process is commonly known as "exporting", as it uses the PyTorch export functionality to convert a PyTorch model into a format suitable for on-device execution. This process yields a .pte file which is optimized for on-device execution using a particular backend.
+
+## Prerequisites
+
+Exporting requires the ExecuTorch python libraries to be installed, typically by running `pip install executorch`. See [Installation](getting-started.md#Installation) for more information. This process assumes you have a PyTorch model, can instantiate it from Python, and can provide example input tensors to run the model.
+
+## The Export and Lowering Process
```

Comment:
Can we cross-reference to this page/diagram?

https://pytorch.org/executorch/main/getting-started-architecture.html

Reply:
Let's chat more about this as a follow-up, if that's okay. The thing I don't love about that diagram is that it's almost a full page long and potentially intimidating. I would like to include more diagrams, but I wonder if we can create a simplified one for the getting started / high-level documentation.